### DISCUSSION TITLE
CORS Headers & Cross Site Embedding

### DISCUSSION THREAD

**Discussion by Elliot-Construct** [Budibase/budibase#4853] [Help]
Hey there Budi-s!

I am confused as to the location of the Web Service Configuration files are; and specifically the CORS Headers of the docker-compose version of Budibase so I can enable the embedding of my apps within a portal container system.

I'd really appreciate any knowledge people can impart about it¬! 

Cheers!
Elliot

**Comment by Elliot-Construct** [comment]
So I have found a solution by myself:

- [x] Bash into the bbproxy container
- [x] cd into etc -> then cd nginx
- [x] then apt-get update to update the repos
- [x] install a command line text editor such as nano (apt-get install nano)
- [x] edit nginx.conf (nano nginx.conf)
- [x] Find the section called Security Headers and customise to suit your CORS security needs.

Cheers
Elliot

**Reply by mjashanks** [reply]
Thanks for reporting back @Elliot-Construct !
